core/templates/dev/head/services/ContextService.ts

```diff
@@ -84,7 +84,8 @@ oppia.factory('ContextService', [
             } else if (pathnameArray[i] === 'question_editor') {
               pageContext = PAGE_CONTEXT.QUESTION_EDITOR;
               return PAGE_CONTEXT.QUESTION_EDITOR;
-            } else if (pathnameArray[i] === 'practice_session') {
+            } else if (pathnameArray[i] === 'practice_session' ||
```

Comment:
When something in parens spans multiple lines, break at '(':

```
    } else if (
        pathnameArray[i] === 'practice_session' ||
        pathnameArray[i] === 'review_test') {
      ...
    }

Reply:
Done.